webapp/package.json

```diff
@@ -17,6 +17,7 @@
     "babel-core": "^6.26.3",
     "babel-loader": "^7.1.4",
     "babel-preset-es2015": "^6.24.1",
+    "chokidar": "^3.5.3",
```

Comment:
What is this library for ?

Reply:
@rdenarie It is used for watching the changes on Vue files. it was missing and I was asked to install it when running ```npm run watch```